conbench/entities/benchmark_result.py

```diff
@@ -388,18 +394,39 @@ class BenchmarkResultSerializer:
 class _BenchmarkFacadeSchemaCreate(marshmallow.Schema):
     run_id = marshmallow.fields.String(
         required=True,
-        metadata={"description": "Unique identifier for a run of benchmarks."},
+        metadata={
+            "description": conbench.util.dedent_rejoin(
+                """
+                Identifier for a Run. This can be the ID of a known Run (as
+                returned by /api/runs) or a new ID in which case a new Run
+                entity is created in the database.
+                """
+            )
+        },
     )
     run_name = marshmallow.fields.String(
         required=False,
         metadata={
-            "description": "Name for the run. When run in CI, this should be of the style '{run reason}: {commit sha}'."
+            "description": conbench.util.dedent_rejoin(
+                """
+                Name for the run. When run in CI, this should be of the style
+                '{run reason}: {commit sha}'. Ignored when run was previously
+                created.
+                """
+            )
         },
     )
     run_reason = marshmallow.fields.String(
         required=False,
         metadata={
-            "description": "Reason for run (commit, pull request, manual, etc). This should be low cardinality. 'commit' is a special run_reason for commits on the default branch which are used for history"
+            "description": conbench.util.dedent_rejoin(
+                """
+                Reason for run (commit, pull request, manual, etc). This should
+                be low cardinality. 'commit' is a special run_reason for
+                commits on the default branch which are used for history.
```

Comment:
oh shoot can you remove
>'commit' is a special run_reason for commits on the default branch which are used for history

That's no longer true 🙂

Reply:
lovely feedback, thanks